.github/workflows/google_java_format.yml

```diff
@@ -3,8 +3,8 @@ name: Format
 
 on:
   pull_request:
-    branches:
-      - master
+    branches: [master]
+    types: [edited, opened, reopened]
```

Comment:
Dunno why these changes come up

Reply:
This is fine, but if you want the pr to be clean of old commits, i think you can merge origin master into the branch.

Reply:
never mind, we can merge this and close the other pr?